common/src/main/java/org/geysermc/floodgate/util/FileUtils.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2019-2021 GeyserMC. http://geysermc.org
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * @author GeyserMC
+ * @link https://github.com/GeyserMC/Floodgate
+ */
+
+package org.geysermc.floodgate.util;
+
+import java.io.InputStream;
+
+public class FileUtils {
+
+    /**
+     * Get an InputStream for the given resource path
+     *
+     * @param resource Resource to get
+     * @return InputStream of the given resource
+     */
+    public static InputStream getResource(String resource) {
+        return FileUtils.class.getClassLoader().getResourceAsStream(resource);
```

Comment:
The geyser version of this had an [AssertionError](https://github.com/GeyserMC/Geyser/blob/a5a849c0598566744b5e08724a5ff849c88b9636/connector/src/main/java/org/geysermc/connector/utils/FileUtils.java#L154) if the stream was null but I'm not really sure how that should be done here